### PR TITLE
Association types / taxons support

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

File: src/Denormalizer/AssociationTypeCreatedDenormalizer.php [hunk 1]
```diff
@@ -13,26 +13,18 @@
...
     /**
      * {@inheritdoc}
      */
     public function denormalize(AMQPMessage $message)
     {
-        $decodedMessage = json_decode($message->getBody(), true);
+        $payload = json_decode($message->getBody(), true)['payload'];
 
-        return new AssociationTypeCreated($decodedMessage['code'], $decodedMessage['labels']);
+        return new AssociationTypeCreated($payload['code'], $payload['labels']);
```
Comment (lchrusciel) [inline src/Denormalizer/AssociationTypeCreatedDenormalizer.php:28]: Is it possible to doesn't have this keys in payload?

File: src/Resources/config/services.xml [hunk 2]
```diff
@@ -13,25 +13,28 @@
...
             <tag name="event_subscriber" subscribes_to="Sylake\SyliusConsumerPlugin\Event\TaxonCreated" method="handleTaxonCreated" />
         </service>
 
         <service id="sylake_sylius_consumer.denormalizer.association_type" class="Sylake\SyliusConsumerPlugin\Denormalizer\AssociationTypeCreatedDenormalizer">
             <tag name="rabbitmq_simplebus.amqp_denormalizer" />
         </service>
-        <service id="sylake_sylius_consumer.denormalizer.product" class="Sylake\SyliusConsumerPlugin\Denormalizer\ProductCreatedDenormalizer">
-            <tag name="rabbitmq_simplebus.amqp_denormalizer" />
-        </service>
+        <!--<service id="sylake_sylius_consumer.denormalizer.product" class="Sylake\SyliusConsumerPlugin\Denormalizer\ProductCreatedDenormalizer">-->
```
Comment (lchrusciel) [inline src/Resources/config/services.xml:35]: :(

File: src/Denormalizer/AssociationTypeCreatedDenormalizer.php (hunk 1 above)
Comment (pamil) [reply src/Denormalizer/AssociationTypeCreatedDenormalizer.php:28]: Tests incoming, thinking of introducing OptionsResolver here in the next iteration.

File: src/Resources/config/services.xml (hunk 2 above)
Comment (Arminek) [inline src/Resources/config/services.xml:35]: 🌮

Comment (pamil) [reply src/Resources/config/services.xml:35]: Minimal effort just to make *something* work 🎉

**Review by Arminek — APPROVED** [review]
👍

**Pull request merged**